src/libs/ReportActionsUtils.js

```diff
@@ -402,6 +402,16 @@ function isCreatedTaskReportAction(reportAction) {
     return reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT && _.has(reportAction.originalMessage, 'taskReportID');
 }
 
+/**
+ * A helper method to identify if the message is deleted or not.
+ *
+ * @param {Object} reportActions
+ * @returns {Boolean}
+ */
+function isMessageDeleted(reportActions) {
```

Comment:
Can't we use this function which already exists -
https://github.com/Expensify/App/blob/2386491070a6b8d111c170a4fc9d1cb288842a3c/src/libs/ReportActionsUtils.js#L45-L49

Also, your parameter should be named `reportAction` and not `reportActions` in case we do end up adding this function.

Apologies, if contributors can't comment on other PRs but I was working on a similar issue and came here to see what was being changed here.

Reply:
Cool, adding a new function makes sense but since the parameter is a single action, I thought it should be named `reportAction` and in most places it is singular only but that is just my suggestion so it is up to you and the reviewers to decide.